components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -119,6 +143,13 @@ export default function TeamSettings() {
                     >
                         Update Organization
                     </Button>
+
+                    <CheckBox
```

Comment:
**suggestion**: This section definitely needs to have a section header, as it's unrelated to the org details above, and it's disconnected from the form above. Using `h2` and `pt-12` for top padding as the section below should suffice. I'll add a before and after design in the review comment. I'd go with a heading like the following to match our docs":

> Collaboration & Sharing

Later on, we could group org settings better and the org settings grow, see [relevant discussion](https://linear.app/gitpod/issue/UX-3#comment-b43c321d) (internal).